astro/src/content/blog/identity-verification-before-registration.mdx

```diff
@@ -0,0 +1,296 @@
+* Lambda HTTP Connect, to make an API call
+
+You'll also use the [Fideo Identity Verification API](https://docs.fideo.ai/docs/verify).
+
+The prerequisites:
+
+* NodeJS 22.x or newer
+* an Essentials or Enterprise license key, which you can [purchase here](/pricing); after purchase you will find it in [your Account Portal](https://account.fusionauth.io/account/plan/)
+* An API key from Fideo, which you can get by [contacting them](https://www.fideo.ai/contact-us/)
+
+<Aside type="note">
+The approach used here works with any other identity verification systems that have a REST API, but you're going to use Fideo for this tutorial.
+</Aside>
+
+Let's take a look at how the Fideo identity verification API works.
+
+## The Fideo Identity Verification API
+
+With this API, you provide as much or as little information about the user as you have. You must have either an email address or a phone number, but you can also add the user's address or name as well.
+
+Here's an example API call using curl.
+
+```shell
+curl -X POST \
+  https://api.fideo.ai/verify \
+  -H 'Authorization: Bearer APIKEY' \
+  -H "Content-Type: application/json" \
+  -d '{
+  "emails": [ "test@example.com" ]
+}'
+```
+
+The API returns JSON similar to this:
+
+```
+{
+  "risk": 0.04188683684800404,
+  "checks": [
+    {
+      "id": "BREACHED_IDENTITY_COMPROMISED",
+      "state": "LOW",
+      "name": "Identity Compromised 30 days or more ago",
+      "description": "The identity has been compromised 30 days or more ago, which could indicate a low potential fraud attempt.",
+      "risk": "LOW",
+      "checkPackage": "Breached"
+    }, 
+     ...  many more checks
+  ],
+  "referenceId": "0197ecce-9332-7e4d-9f49-a648b3ca807f"
+}
+```
+
+Consult [the Fideo docs](https://docs.fideo.ai/docs/verify#api-output) for details about these fields. Contact them directly to discuss what makes sense for your use case. A KYC flow for a bank would use different checks and logic than an online gaming company trying to keep bots out.
+
+For the purpose of this tutorial, the logic will rely heavily on the top level `risk` value. The `risk` is a number between 0 and 1, with 0 being safe and 1 being as risky as it gets.
+
+Let's head back to set up the FusionAuth system to integrate with this API using an example application.
+
+## Set Up FusionAuth
+
+First, you need to get FusionAuth running and configured.
+
+The simplest way to run FusionAuth is to install it on your development machine (which is probably the laptop you are reading this on). Pick your preferred method of installation below. 
+
+<DownloadWidget kickstartEnabled={true} kickstartName={"identity-verification"} kickstartCustomizationInstructions={["Replace LICENSE_KEY with the FusionAuth license key.","Replace FIDEO_API_KEY with your Fideo API Key."]} />
+
+Kickstart will set up FusionAuth with the correct configuration for this use case. This includes:
+
+* a sample Changebank application with a custom look and feel
```

Comment:
Just as a note, this will likely change in the not-so-distant future to remove Changebank and replace with something simpler and more FusionAuth-branded, so mentions like this will need to be updated. In preparation for such potential changes, it might be good to just remove the word "Changebank" so updates later aren't required.

Reply:
acknowledged.